pages/api/fade.md

```diff
@@ -23,7 +23,7 @@ It uses [react-transition-group](https://github.com/reactjs/react-transition-gro
 | <span class="prop-name">in</span> | <span class="prop-type">bool</span> |  | If `true`, the component will transition in. |
 | <span class="prop-name">timeout</span> | <span class="prop-type">union:&nbsp;number&nbsp;&#124;<br>&nbsp;{ enter?: number, exit?: number }<br></span> | <span class="prop-default">{  enter: duration.enteringScreen,  exit: duration.leavingScreen,}</span> | The duration for the transition, in milliseconds. You may specify a single timeout for all transitions, or individually with an object. |
 
-The component cannot hold a ref.
+The `ref` is forwarded to the root element.
```

Comment:
Hmm 😕

Reply:
We need more control over this description. I'm not even sure this is correct for the other components. Some JSDOC block before the `describeConformance` might be the best place for meta information. I have some better ref tests locally. Will think of something.